### PR TITLE
[EA] Add countdown to frontpage vote banner

### PR DESCRIPTION
**Pull request by Will-Howard**
Screenshot 2023-12-11 at 15 06 31
Screenshot 2023-12-11 at 15 07 18

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206142485611766) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonBanner.tsx [hunk 1]
```diff
@@ -291,13 +307,43 @@ const BannerDate: FC<{
   </div>
 );
 
+/**
+ * Convert from e.g. 1d to "1 day"
+ */
+function convertToLongFormat(time: string): string {
```
Comment (Will-Howard) [inline packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonBanner.tsx:313]: I couldn't find a good place to put this

Comment (jpaddison3) [reply packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonBanner.tsx:313]: `packages/lesswrong/lib/utils/timeUtil.ts`

**Review by jpaddison3 — APPROVED** [review]
Code is ASTLAC, I would love it if @agnesstenlund could adjust some pixels on the desktop view, but seems good enough given that she's out.

image

File: packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonBanner.tsx (hunk 1 above)
Comment (Will-Howard) [reply packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonBanner.tsx:313]: Oh nice, I must have only looked for a `dateUtil`

**Comment by Will-Howard** [comment]
I'm going to merge this seeing as Agnes is out and it would be good to show it as soon as possible

**Pull request merged**